src/main/java/com/toamistaketracker/ToaMistakeTrackerConfig.java

```diff
@@ -23,9 +24,444 @@ default boolean showMistakesInChat() {
             name = "Show Mistakes On Overhead Text",
             description = "When a player makes a mistake in ToA, whether or not to show the mistake message above " +
                     "their head as overhead text.",
-            position = 2
+            position = 1
     )
     default boolean showMistakesOnOverheadText() {
         return true;
     }
+
+    @ConfigItem(
+            keyName = "stackQuestionMarks",
+            name = "Stack Mistake Messages",
+            description = "When a mistake message is set to \"?\", whether or not to stack the message based on " +
+                    "the total number of mistakes made.",
+            position = 2
+    )
+    default boolean stackQuestionMarks() {
+        return true;
+    }
+
+    @ConfigItem(
+            keyName = "stackOnlySameMistakes",
+            name = "Stack Only Same Mistakes",
+            description = "When \"Stack Mistake Messages\" is toggled on, whether or not the amount of times the " +
+                    "mistake message stacks is based on the current mistake made or all raid mistakes.",
+            position = 3
+    )
+    default boolean stackOnlySameMistakes() {
+        return true;
+    }
```

Comment:
> or all raid mistakes

Is this for the current raid? For example, if akkha orb mistakes so far is 2, but total raid mistakes (this raid) is 10, this would prepend 2 `?`s if this is true, and 10 `?`s if this is false, right? 

I guess my initial confusion was if this was for "all raid mistakes" ever as opposed to current raid. Wonder if there's a better way to get this across. Is this a feature you're particularly interested in, or just thought it could be neat to add?

Reply:
This might actually work better as an enum with three options, instead of two separate booleans:

`Mistake Message Stacking`:
```
ALL_MISTAKES
SAME_MISTAKE_ONLY [default]
NONE
```

or something along those lines